plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -166,4 +170,18 @@ public IcebergConfig setProjectionPushdownEnabled(boolean projectionPushdownEnab
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    @NotNull
+    public Duration getExpireSnapshotsMinRetention()
+    {
+        return expireSnapshotsMinRetention;
+    }
+
+    @Config(EXPIRE_SNAPSHOTS_MIN_RETENTION)
+    @ConfigDescription("Minimal retention period for expire_snapshot procedure")
+    public IcebergConfig setExpireSnapshotsMinRetention(Duration expireSnapshotsMinRetention)
```

Comment:
is there documentation you would like to extend already?

Reply:
I am not aware of any documentation but I may be missing something.